bokeh/plotting/tests/test_helpers.py

```diff
@@ -1,7 +1,13 @@
 import datetime
 from mock import mock
-
-import pandas as pd
+from unittest import skipIf
+
+import numpy as np
+try:
+    import pandas as pd
+    is_pandas = True
+except ImportError as e:
+    is_pandas = False
```

Comment:
FWIW I'd personally be fine just stipulating that pandas is a required *test* dependency and not complicating the test files like this but I don't have super strong feelings about it, either

Reply:
For me it is also fine either way. This was the style in test_sources.py, which is why I copied it. I think you should just decide on one style, and then I can change it everywhere.